napari_aicssegmentation/widgets/collapsible_box.py

```diff
@@ -0,0 +1,98 @@
+from pathlib import Path
+
+from qtpy.QtGui import QPixmap
+from qtpy.QtWidgets import (
+    QFrame,
+    QHBoxLayout,
+    QLabel,
+    QVBoxLayout,
+    QWidget
+)
+
+DIR = Path.cwd() / "napari_aicssegmentation"
+
+
+"""
+A collapsible box widget containing a title box and a content box. The title box can be clicked
+to toggle the visibility of the content box.
+
+Params:
+    step:       Number representing the workflow step. This will be displayed as part of the
+                    title and be used as a part of the object name for identification.
+    title:      String
+    content:    QLayout to be nested inside the content box
+    isOpen:     Boolean, whether the widget is open or collapsed
+    isEnabled:  Boolean, overwrites the built-in property QWidget.enabled property
+"""
+
+
+class CollapsibleBox(QWidget):
+    def __init__(self, step, title, content, isOpen=True, isEnabled=True):
+        super().__init__()
+        self.step = step
```

Comment:
I don't think the concept of a "step" has it's place in the generic Collapsible Box widget. We'll deal with that when we build an actual Workflow Step widget